### ISSUE TITLE
Add build image for RPM packages

### ISSUE THREAD

**Issue by dagood** [dotnet/dotnet-buildtools-prereqs-docker#92]
We have e.g. `ubuntu-14.04-debpkg` for Debian packages, we need something similar to build RPM packages. This would let Core-Setup and others use an actively supported build image when producing RPM packages.

I think we just need to create a new `centos-7-rpmpkg` Dockerfile that extends the CentOS Dockerfile with:

```
RUN yum clean all \
    && yum install -y \
        gcc \
        make \
        rpm-build \
        ruby-devel \
        sudo \
    && gem install --no-ri --no-rdoc fpm \
    && yum clean all
```

This isn't blocking me.

**Comment by MichaelSimons** [comment]
@dagood - Go ahead and create a PR please.  It is expected that the product teams manage the Dockerfiles they need specific to their scenarios.

**Comment by dagood** [comment]
I plan on it when I get a chance to validate it and go through with consuming it in the Core-Setup build infra, or someone from e.g. Core-SDK or ASP.NET Core might want to do this too. Is it bad to open this kind of tracking issue here?

**Comment by MichaelSimons** [comment]
Logging the issue here is great.  I just wanted to clarify expectation with everyone. 😺

**Comment by dagood** [comment]
This is also useful for dev builds, would be worth tackling even if we don't go and *immediately* put it in the official build. /cc @NikolaMilosavljevic

**Comment by MichaelSimons** [comment]
Fixed by #252.
